src/tripCardSample/TripCardSample.story.mdx

```diff
@@ -0,0 +1,32 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { TripCardSample } from '.'
+
+<Meta title="Widgets/TripCardSample" />
+
+# **TripCardSample**
+
+<Canvas>
+  <Story name="Default">
+    <TripCardSample
+      departure="Paris"
+      arrival="Bordeaux"
+      price="8,00 €"
+      priceFromLabel="from"
```

Comment:
```suggestion
      priceLabel="from"
```

Reply:
Done in https://github.com/blablacar/ui-library/pull/1076